API/main.py

```diff
@@ -1,3 +1,5 @@
+from django import db
```

Comment:
Do we need imports from django ????????????

Reply:
It was for the previous method used, changed to this new one so , not needed anymore. (Removed)